README.md

```diff
@@ -94,13 +94,13 @@ On a unix system, the build directory should now look something like this:
 Next, you can run the tests to check the build.
 
 ```bash
-$ make test
+$ cmake --build "build" --config Release --target test
```

Comment:
I have checked the new commands on both Linux and Windows and all seem to work fine (well, the installation failed expectedly with the access rights but looked correct up to that point) except for running tests. I guess it works fine with Makefile (or Ninja) build systems where there is a `test` target. But Visual Studio doesn't have such a target and fails for the new command.

However, it would fail with the old one just as well obviously, so no regress here.

I'm afraid progress could be made here only by using `ctest` instead.

Reply:
why don't we suggest `ctest`?

Reply:
Because we use `enable_testing()` instead of `include(CTest)`. I think it would be better to do the other, but it would go beyond just changing `README.md`. Perhaps as a separate change?

Reply:
works for me :)

Reply:
filed #1023 for this.